app/assets/scss/application.scss

```diff
@@ -22,4 +22,23 @@ body {
 
 img {
   max-width: 100%;
+}
+
+.overview-banner {
+  background-color: #E1F0E9;
+  padding: 20px;
+}
+
+.saved-banner {
+  background-color: #f3f2f1;
+  padding: 0.01rem;
+}
+
+.banner-line {
+  border-top: 0.1rem solid #1d70b8;
+
+}
+
+.contactus-line {
```

Comment:
delete and use section break from GDS

https://design-system.service.gov.uk/styles/typography/